### ISSUE TITLE
Does domain renewal change proof?

### ISSUE THREAD

**Issue by timkgh** [keybase/client#21294]
People who are following me and chatting with me are getting a red alert that some of my proofs have changed, just like in issue #8775

One recent change I can think of is that I renewed a domain that is attached to my proofs, hence changing the domain expiration date.

Is the domain expiration date considered part of the proof?

**Comment by Brianetta** [comment]
No, it isn't. My proofs for ppcis.org have all survived at least one renewal, with a couple of them surviving three.

**Comment by timkgh** [comment]
Wish the warnings were more specific not just "_some_ proofs have changed".

**Comment by sradforth** [comment]
Agreed, it's sort of a useless yet worrying message... someone just mentioned mine changed even though I haven't done anything that I can think of to cause it.  Also randomly getting reset email's from keybase which might or might not be related.
would be good to at least have the technical details of what causes this alert if possible.
